ios/RNFIRMesssaging.m

```diff
@@ -240,10 +240,9 @@ - (void)applicationReceivedRemoteMessage:(FIRMessagingRemoteMessage *)remoteMess
   [_bridge.eventDispatcher sendDeviceEventWithName:FCMNotificationReceived body:[remoteMessage appData]];
 }
 
-RCT_EXPORT_METHOD(presentLocalNotification:(UNNotificationRequest *)data resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+RCT_EXPORT_METHOD(presentLocalNotification:(UNNotificationRequest *)request resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
```

Comment:
@MechanicKim thanks, it's my mistake. 
 Locally I changed all 'data' to 'request', but I create pr via website and here we are :D
I wanted to display notifications when the app was in foreground, but iOS doesn't support this feature (maybe I should add annotation/warning to readme  about iOS behaviour, because it's really different behaviour than android). I didn't test if scheduling work. But I can check that :)